lib/api/service/user_service.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:dio/dio.dart';
+import 'package:retrofit/retrofit.dart';
+import 'package:survey/api/request/login_request.dart';
+import 'package:survey/api/response/login_response.dart';
+
+part 'user_service.g.dart';
+
+@RestApi()
+abstract class UserService {
```

Comment:
I think we should call this one as AuthService, AuthenticationService or OAuthService 🤔